src/server/pfs/server/driver.go

```diff
@@ -1260,6 +1261,9 @@ func (d *driver) makeCommit(
 			if err != nil {
 				return nil, err
 			}
+			defer func() {
+				tree.Destroy()
```

Comment:
This function can return an error, so you should name the returned error parameter and update this code to look like this:

```suggestion
				if err := tree.Destroy(); err != nil && retErr == nil {
				    retErr = err
				}
```